app/uk/gov/hmrc/pillar2submissionapi/validation/ValidationError.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pillar2submissionapi.validation
+trait ValidationError {
+  def errorCode:    String
+  def errorMessage: String
+  def field:        String
+}
+
+object ValidationError {
```

Comment:
I've added these generic errors, mainly for the tests but tbh we probably won't even use these in the actual uktr implementation so i'll remove these